bevy_asset_loader/tests/stageless/can_run_without_next_state.rs

```diff
@@ -0,0 +1,83 @@
+#![allow(dead_code)]
+
+use bevy::app::AppExit;
+use bevy::asset::AssetPlugin;
+use bevy::audio::AudioPlugin;
+use bevy::prelude::*;
+use bevy_asset_loader::{AssetCollection, AssetLoader};
+use iyes_loopless::prelude::*;
+
+#[cfg_attr(
```

Comment:
I think this is already covered with the `cfg` on the `stageless` mod. Same for the other two files in this directory.